agent/server.py

```
@@ -109,70 +109,144 @@ async def consolidate_memory(req: dict[str, Any]) -> dict[str, str]:
             "TODAY'S CORRECTIONS:\n" + json.dumps(corrections, indent=2) + "\n\n"
-            "Output ONLY JSON: {'rules': [...], 'updated_at': '...'}"
+            "Output ONLY JSON: {'rules': [{'action': '...', 'condition': '...', 'added_date': '...', 'last_reinforced_at': '...', 'pinned': bool, 'verified': bool}], 'updated_at': '...'}"
         )
         resp = await llm.ainvoke([("system", sop_prompt), ("human", "Update SOPs.")])
         try:
             new_sops = json.loads(_strip_fences(str(resp.content)))
-            new_sops["updated_at"] = datetime.now(timezone.utc).isoformat()
+            new_sops["updated_at"] = now_str
             await store.aput(ns, "master_sops", new_sops)
             log.info("updated master_sops for %s", clinic_id)
         except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)